src/yacs-web/src/services/ScheduleService.js

```diff
@@ -8,23 +8,33 @@
  * @property {string} time_start
  */
 /**
- * @typedef {Object<string, any>} CourseSection
+ * @typedef {Object} CourseSection
  * @property {string} crn
  * @property {string} department
  * @property {number} level
  * @property {string} semester
  * @property {CourseSession[]} sessions
  */
 /**
- * @typedef {Object<string, any>} Course
+ * @typedef {Object} Course
  * @property {string} department
  * @property {number} level
  * @property {CourseSection[]} sections
  * @property {string} title
  * @property {Date} date_start
  * @property {Date} date_end
+ * @property {string} id
  */
 
+ /**
+  * @typedef {Object} Subsemester
+  * @property {Date} date_start
+  * @property {Date} date_end
+  * @property {string} date_start_display
+  * @property {string} date_end_display
+  * @property {string} text
```

Comment:
More descriptive field name would be nice, e.g. `displayString`

Reply:
Agree will do